manifests/server/config.pp

```diff
@@ -222,21 +226,15 @@
             require => File['systemd-conf-dir'],
         ;
 
-        # Remove old unit file to avoid service failures
+        # Remove old unit file to avoid conflicts
         'old-systemd-override':
             ensure => absent,
             path   => "/etc/systemd/system/${service_name}.service",
         ;
       }
-
-      exec { 'restart-systemd':
```

Comment:
To be clear, this should be kept in since the module still supports Puppet 5, but you can guard it on a version fact.

Reply:
I'll change this to use the $puppetversion fact.